scss/utilities/_json-api.scss

```diff
@@ -3,6 +3,10 @@
 
 @use 'json-encode' as json;
 
+////
+/// @todo add `@use '~/sassdoc-theme-herman/scss/utilities';` at the top of every file you intend to use this.
+////
+
```

Comment:
Why are these given as `@todo` annotations? That's generally used for features we intend to add. This should be part of the main documentation prose instead. But I would avoid specific paths - that differs from project to project - and instead clarify generally that the module needs to be used.